src/pkg/caendr/caendr/api/phenotype.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from caendr.models.datastore import Species
+from caendr.models.error import BadRequestError
+from caendr.models.sql import PhenotypeMetadata
+from caendr.services.cloud.postgresql import rollback_on_error
+
+
+@rollback_on_error
```

Comment:
consider removing the rollback -- `@rollbacks` are great when there are side-effects to the DB calls and we want to bundle them into a transaction in order to undo the changes in case something goes wrong. In this case, it appears that we are just `reading` (query.all) and wouldn't get into the scenario where we would need to rollback a change to the db.